CMakeLists.txt

```diff
@@ -72,11 +73,50 @@ include(cmake/secp256k1.cmake)
 include(CheckStdFilesystem)
 check_std_filesystem()
 
+find_package(PkgConfig)
+if(BUILD_DAEMON)
+  set(THREADS_PREFER_PTHREAD_FLAG ON)
+  find_package(Threads REQUIRED)
+  if(NOT MINGW AND NOT CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+    target_compile_definitions(Threads::Threads INTERFACE
+      $<$<COMPILE_FEATURES:cxx_thread_local>:HAVE_THREAD_LOCAL>
+    )
+  endif()
+
+  if(APPLE)
+    execute_process(
+      COMMAND brew --prefix boost
```

Comment:
Is there a check for `brew`, like we currently do `AC_CHECK_PROG`?

Reply:
Thanks! [Updated](https://github.com/hebasto/bitcoin/pull/7#issuecomment-1443742611).